weblate_web/static/style.css

```diff
@@ -4102,3 +4102,121 @@ input.fullwidth {
     font-size: 10px;
   }
 }
+
+/*Testimonial and Slider CSS*/
+
+.testimonial{
+  width: 365px;
+  border-radius: 30px;
+  background-color: #ffffff;
+  box-shadow: 0 2px 10px 0 rgba(0,0,0,0.1);
+  margin-left: 10px;
+}
+
+.icon-review{
+  text-align: left;   
+}
+.icon-review img{
+  margin-left: 50px;
+}
+.icon-review p{
+  margin-left:35px;
+  margin-top: -25px;
+  width: 70%;
+  font-size: 15px;
+  line-height: 18px;
+  letter-spacing: 0.5px;
+  color:#2A3744 ;
+}
+
+.circle-user{
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  position: relative;
+  left: 32px;
+  margin-top: 20px;
+}
+.circle-user img{
+  margin: 0px;
+  border-radius: 50%;
+}
+
+.icon-user{
+  display: flex;
+  text-align: left;
+  padding-bottom: 25px;
+}
+.contentt{
+  margin-top: 20px;
+  margin-left: 45px;
+}
+.contentt a{
+  color: #1FA385;
+  line-height: 23px;
+  text-decoration: underline;
+}
+.unqimg{
+  margin: 0px !important;
+}
+
+.slider-slide {
+display: -webkit-flex;
+display: flex;
+-webkit-align-items: center;
+align-items: center;
+-webkit-justify-content: center;
+justify-content: center;
+}
+
+.container,
+.footer,
+.main,
+.header {
+width: 100%;
+max-width: 1000px;
+margin-left: auto;
+margin-right: auto
+}
+
+@media screen and (min-width: 600px) {
+.main {
+  display: -webkit-flex;
+  display: flex
+}
+}
```

Comment:
Are these styles used? I could not find single use of "main" class.

Reply:
I have removed the useless css.